functions.sh

```diff
@@ -88,7 +92,11 @@ upgrade() {
 # Returns:
 #     None
 install_packages() {
-    chroot_exec apt-get -y ${PM_OPTIONS} install $*
+    if [ ! -z ${ENABLE_UNATTENDED_INSTALLATION} ]; then
```

Comment:
`ENABLE_UNATTENDED_INSTALLATION` is a boolean var, so let's simplify the condition the following way: 
```if ${ENABLE_UNATTENDED_INSTALLATION}; then```